docs/testing/application-specific-api-error-handling.md

```diff
@@ -64,7 +61,7 @@ The following table is a list of errors that the API may return.
 |`AccessDenied` |You cannot perform the requested operation.| |
```

Comment:
Are these errors returned in applications other than Excel? About half of the descriptions use Excel-specific language (workbook, worksheet, range, PivotTable, etc) which I think is strange for a page that is supposed to have errors for all application-specific APIs. Maybe we divide the errors into separate tables? Such as: One table for application-neutral, one for Excel only, and one for the other applications if they have any application-specific errors

Reply:
I'm torn here. I agree that mixing Excel-specific errors with more general ones is confusing. However, all of these were listed as "Excel" errors, despite also being thrown in Word (and beyond?). To me, this indicates these were Excel-only at some point and have been adopted elsewhere. That's a pattern that could continue. So, rather than having an ever-shifting list or table of which errors can come from which hosts, I think it's easier in the long run to have a main list of app-specific errors.

Put another way, the purpose of this table is as a lookup for when someone gets an error. They see something get thrown, copy and paste the error message into search, land on our page, and scroll through the table. The text surrounding what they're looking doesn't necessarily matter.

Anyway, that's my thoughts.